index.html

```diff
@@ -369,7 +372,7 @@ <h3>
             <dd>
               <p>
                 Takes no arguments. Updates any other parts of the implementation that need to be
-                kept in sync with changes in the results of <a>permission states</a> or [=powerful
+                kept in sync with changes in the results of permission [=permission/states=] or [=powerful
```

Comment:
The fact that you're keeping "permission" here and in several other places seems like a sign that the term should stay "permission state", and not be just "state".

Reply:
Yeah, I was thinking the same. I might alias it.